src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/InertContentType.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
```

Comment:
```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

﻿using System;
```

Reply:
@TanayParikh / @NTaylorMullen have you guys considered putting this in your [.editorconfig file](https://docs.microsoft.com/en-us/visualstudio/ide/reference/add-file-header?view=vs-2019#how-to) so that the IDE recommends it as a code fix?

```
file_header_template = // Copyright (c) .NET Foundation. All rights reserved.\n// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Reply:
O_O noooo definitely have not considered that but that sounds amazing. We should definitely do that! I've filed an issue: https://github.com/dotnet/aspnetcore/issues/23812

I'll make sure to keep adding new pieces to the issue for when we add that.

Reply:
Added, thanks!

---
In reply to: [451968839](https://github.com/dotnet/aspnetcore-tooling/pull/2201#discussion_r451968839) [](ancestors = 451968839)